pulsar-client/src/main/java/org/apache/pulsar/client/impl/TypedMessageBuilderImpl.java

```diff
@@ -84,7 +84,7 @@ public MessageId send() throws PulsarClientException {
                 producer.triggerFlush();
             }
 
-            return sendFuture.get();
+            return sendFuture.get(Integer.MAX_VALUE, TimeUnit.MILLISECONDS);
```

Comment:
Please use `sendTimeoutMs`

Reply:
> Please use `sendTimeoutMs`

Maybe needn't. If timeout, the future will throw exception automatically.

Reply:
> Please use `sendTimeoutMs`


Do you mean to modify producter sendTimeoutMs configuration? This PR is to avoid `CompletableFuture#get()` performance issues in some scenarios. In 8u, `CompletableFuture#get()` calls `Runtime.availableProcessors` during spin in waitingGet. `Runtime.availableProcessors` performance is not good , [link](https://bugs.openjdk.java.net/browse/JDK-8227006). So use `CompletableFuture#get(long, TimeUnit)` instead.

Reply:
> > Please use `sendTimeoutMs`
> 
> Maybe needn't. If timeout, the future will throw exception automatically.

Yes, right.

Reply:
> > Please use `sendTimeoutMs`
> 
> Do you mean to modify producter sendTimeoutMs configuration? This PR is to avoid `CompletableFuture#get()` performance issues in some scenarios. In 8u, `CompletableFuture#get()` calls `Runtime.availableProcessors` during spin in waitingGet. `Runtime.availableProcessors` performance is not good , [link](https://bugs.openjdk.java.net/browse/JDK-8227006). So use `CompletableFuture#get(long, TimeUnit)` instead.

How about add a short comment with link above this line ?

Reply:
agree, I will add a comment.

Reply:
@haoyann Let's also add this comment to the PR description.